doc/en/user/source/styling/sld-extensions/pointsymbols.rst

```diff
@@ -304,6 +288,35 @@ For example:
 
 In this example an SVG graphic is being used, so the size is specified explicitly. 
 
+Bulk WKT Shapes
+~~~~~~~~~~~~~~~
+
+It is possible to create a symbol set of your own custom marks using a property file.
+
+Here is an :file:`example.properties`:
+
+.. code-block:: text
+   
+   zig=LINESTRING(0.0 0.25, 0.25 0.25, 0.5 0.75, 0.75 0.25, 1.00 0.25)
+   block=POLYGON((0 0, 1 0, 1 1, 0 1, 0 0))
+
+The SLD to use the symbols defined in :file:`example.properties` is:
+
+.. code-block:: xml
+   :linenos: 
+
+    <PointSymbolizer>
+      <Graphic>
+        <ExternalGraphic>
```

Comment:
This looks definitely wrong, WKTMarkFactory emits a shape, how is one going to assign stroking and filling to it? I'm for one surprised this works at all, but if it does, the code needs fixing, not the documentation. The previous documentation correctly documents the intended behavior and usage, there might be issues with relative paths not being recognized.